scipy/optimize/slsqp.py

```diff
@@ -9,7 +9,6 @@
     'OptimizeResult',
     'fmin_slsqp',
     'slsqp',
-    'zeros',
```

Comment:
Yep this looks like the right fix :+1:

Reply:
Ah thanks, I found it after the fact. But good to have confirmation.